store/postgres/tests/relational.rs

```diff
@@ -187,8 +187,13 @@ fn insert_entity(conn: &PgConnection, layout: &Layout, entity_type: &str, entity
         entity_type.to_owned(),
         entity.id().unwrap(),
     );
-    let errmsg = format!("Failed to insert entity {}[{}]", entity_type, key.entity_id);
-    layout.insert(&conn, &key, entity, 0).expect(&errmsg);
+    let entity_id = key.entity_id.clone();
+    let entity_type = EntityType::from(entity_type);
+    let mut entities = vec![(key, entity)];
+    let errmsg = format!("Failed to insert entity {}[{}]", entity_type, entity_id);
+    layout
+        .insert(&conn, &entity_type, &mut entities, 0)
+        .expect(&errmsg);
```

Comment:
It would be good to also test that inserting 2 or 3 entities work, just to have a test that our SQL generation doesn't break in the future.

Reply:
Additionally, I think we could use an `assert` right into this helper function, so we are also testing that the expected number of entities were inserted, updated or deleted (according to the specific function).